packages/web-pkg/src/instance/index.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { EventBus } from '../event'
```

Comment:
File is not in the right location (needs to be in the `tests/unit` folder hierarchy)